README.md

```diff
@@ -123,6 +124,7 @@ the individual Load Balancer types.
 | [Dualstack support][dualstack] | :heavy_check_mark: `--ip-addr-type=dualstack` | :heavy_multiplication_x: |
 | [Idle Timeout][idle_timeout] | :heavy_check_mark: `--idle-connection-timeout` | :heavy_multiplication_x: |
 | Custom Security Group | :heavy_check_mark: | :heavy_multiplication_x: |
+| HTTP/2 Support | :heavy_check_mark: | (not relevant) |
```

Comment:
maybe better to have a :weak_check_mark: , because ALBs convert H2 to HTTP/1.1 to call the backends

Reply:
Made it :white_check_mark: instead of :heavy_check_mark: :)